### ISSUE TITLE
Improve the Donation Page of the Application 

### ISSUE THREAD

**Issue by N-45div** [N-45div/Resilient-Response#20]
Description : 
Give the User Freedom to pay with paypal or anyother payment method

**Comment by DeexithParand2k2** [comment]
Hi @N-45div can I take up this issue please. Can you please assign this to me.

**Comment by N-45div** [comment]
Yes @DeexithParand2k2 you can take up this issue buddy and start doing it...

**Comment by DeexithParand2k2** [comment]
Sure I'll work on it and let you know once I complete it.

**Comment by deexithparand** [comment]
Hi @N-45div until now I've added new components and separated donation into two components (one for gift donations and another for monetary donations ). Now for the payment gateway issue, some of the payment gateway integration require backend also, so is it fine if I add node.js backend to handle the payment part alone.

**Comment by N-45div** [comment]
> Hi @N-45div until now I've added new components and separated donation into two components (one for gift donations and another for monetary donations ). Now for the payment gateway issue, some of the payment gateway integration require backend also, so is it fine if I add node.js backend to handle the payment part alone.

While integrating the backend take care that other components/pages don't get disturbed that's it :) ...Well done ahead...
